versions/1.1/schema/digitized_audioreeldigital.json

```diff
@@ -260,8 +260,9 @@
               "enum": [
                 "stereo",
                 "mono",
-                "unknown",
-                "multitrack"
+                "multitrack",
+                "multi-track",
```

Comment:
It looks like multitrack was already an accepted spelling in this schema. If we've been accepting multitrack already, I think the other schema should use that spelling too.